sympy/solvers/inequalities.py

```diff
@@ -982,3 +986,297 @@ def reduce_inequalities(inequalities, symbols=[]):
+        if type(rel) == Ne:
+            raise TypeError("'not equal to' is not allowed in linear programming.")
+        if not rel.rhs.is_constant():
+            raise ValueError("Right hand side of relation must be constant")
+
+        variables = variables | rel.free_symbols
+
+        if type(rel) == Le:
+            eqns.append(rel.lhs - rel.rhs)
+        elif type(rel) == Ge:
+            eqns.append( rel.rhs - rel.lhs)
+        elif type(rel) == Eq:
+            # x = 3 can be represented as x <= 3 and x >= 3
+            eqns.append(rel.lhs - rel.rhs)
+            eqns.append(rel.rhs - rel.lhs)
+        else:
+            raise TypeError(f"Unrecognized relational: {rel}")
+
+    variables = sorted(variables, key=lambda v: str(v)) # order symbols lexicographically
+
+    A, B = linear_eq_to_matrix(eqns, *variables)
+    C = linear_eq_to_matrix(objective, *variables)[0] # constant terms can be safely ignored here
+
+    return A, B, C
+
+# Maximize Cx + D constrained with Ax <= B and x >= 0
+# Minimize y^{T}B constrained with y^{T}A >= C^{T} and y >= 0
+def linear_programming(constraints, objective):
+    """
+    A function to maximize a linear objective function subject to linear
+    constraints with the simplex method.
+
+    Explanation
+    ===========
+
+    When x is a column vector of variables, y is a column vector of dual variables,
+    and when the objective is either:
+
+    *) Maximizing Cx constrained to Ax <= B and x >= 0.
+    *) Minimizing y^{T}B constrained to y^{T}A >= C^{T} and y >= 0.
+
+    The method returns a triplet of solutions optimum, argmax and argmax_dual where
+    optimum is the optimum solution, and argmax/ argmax_dual give the
+    values for x and y, respectively.
+
+    Parameters
+    ==========
+
+    constraints : list of linear inequalities
+        The lhs of these inequalities are equivalent to the matrix A and the rhs
+        are equivalent to the column vector B.
+
+    objective : a linear function to maximize
+        The objective function is represented by the row vector C.
+
+    random_seed : int
+        Determines the random pivoting behavior of the simplex method
+
+    Examples
+    ========
+
+    >>> from sympy.abc import x, y, z
+    >>> from sympy.solvers.inequalities import linear_programming
+    >>> r1 = y+2*z <= 3
+    >>> r2 = -x-3*z <= -2
+    >>> r3 = 2*x+y+7*z <= 5
+    >>> optimum, argmax, argmax_dual = linear_programming([r1,r2,r3], x+y+5*z)
+    >>> optimum
+    11/3
+    >>> argmax
+    [0, 1/3, 2/3]
+    >>> argmax_dual
+    [0, 2/3, 1]
+
+    The method also works with non-integer coefficients (like sqrt(2)).
+
+    >>> from sympy import sqrt
+    >>> r1 = y + sqrt(2)*z <= sqrt(3)
+    >>> r2 = -x-3*z <= -2
+    >>> r3 = 2*x+y+7*z <= 5
+    >>> optimum, argmax, argmax_dual = linear_programming([r1,r2,r3], x+y+sqrt(5)*z)
+    >>> optimum
+    3
+    >>> argmax
+    [2, 1, 0]
+    >>> argmax_dual
+    [0, 1, 1]
+
+    See Also
```

Comment:
Is there a way to link to SciPy's linear programing function? 
https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.linprog.html

Reply:
You can use the `external` syntax (see for example the `plot` docstring). I'm not sure if that works in see also or not. You'll have to test it.